packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -0,0 +1,151 @@
+import {IDesignSystem, hexToRGB} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IToggleClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = {
+    toggle: {
+        '& > label': {
+            display: 'inline-block',
+            fontSize: '13px',

```

Comment:
Let's use the type ramp config here.

Reply:
Sure - having trouble figuring out how to do it exactly... Example?